apps/omg_api/lib/root_chain_coordinator.ex

```diff
@@ -53,17 +55,18 @@ defmodule OMG.API.RootchainCoordinator do
   end
 
   def handle_call({:check_in, synced_height, service_name}, {pid, _}, state) do
-    {:ok, state} = Core.check_in(state, pid, synced_height, service_name)
+    {:ok, state, services_to_sync} = Core.check_in(state, pid, synced_height, service_name)
+    Enum.each(services_to_sync, fn pid -> send(pid, :sync) end)
```

Comment:
It seems that for every ethereum height, service can get from 1 to N-1 :sync messages, where N is number of services.

Reply:
That's not the case.